protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/toconnectschema/ProtobufSchemaToConnectSchemaConverter.java

```diff
@@ -116,8 +120,15 @@ private SchemaBuilder toConnectSchemaBuilderForField(final Descriptors.FieldDesc
 
         //Protobuf provides different types of integers.
         //We add metadata to Connect schema to store the original type used.
+        //final SchemaBuilder schemaBuilder = new SchemaBuilder(connectType);
         if (TYPES_TO_ADD_METADATA.contains(protobufType)) {
             schemaBuilder.parameter(PROTOBUF_TYPE, protobufType.name().toUpperCase());
+        } else if (protobufType.equals(ENUM)) { //ENUM case; storing ENUM data as metadata to avoid being lost in translation.
+            schemaBuilder.parameter(PROTOBUF_TYPE, PROTOBUF_ENUM_TYPE);
+            for (Descriptors.EnumValueDescriptor  enumValueDescriptor: fieldDescriptor.getEnumType().getValues()) { //iterating through the values of the Enum to store each one
```

Comment:
NIT: extra space between `Descriptors.EnumValueDescriptor` and `enumValueDescriptor`